docs/src/integration-keda.md

```
@@ -0,0 +1,46 @@
+---
+title: Integrations with KEDA
+---
+
+
+
+# Integration with KEDA
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)